modules/appeals_api/app/models/appeals_api/notice_of_disagreement.rb

```diff
@@ -164,14 +173,41 @@ def add_missing_claimant_info_error(field, attribute_name:)
       errors.add attribute_name, "if any claimant info is present, claimant #{field} must also be present"
     end
 
+    def validate_hearing_type_selection
+      return if board_review_hearing_selected? && includes_hearing_type?
+
+      if hearing_type_missing?
+        errors.add :form_data, "if 'boardReviewOption' 'hearing' is selected, a 'hearingType' must be selected"
```

Comment:
Let's give the full path to the `boardReviewOption` and `hearingType` attributes like we do for `#validate_that_at_least_one_set_of_contact_info_is_present`

Reply:
Done! Much better thanks.